pke/kyber/internal/common/poly_test.go

```diff
@@ -27,8 +27,8 @@ func TestDecompressMessage(t *testing.T) {
 	var m, m2 [PlaintextSize]byte
 	var p Poly
 	for i := 0; i < 1000; i++ {
-		_, err := rand.Read(m[:])
-		if err != nil {
+		n, err := rand.Read(m[:])
+		if err != nil || n != len(m) {
 			t.Error(err)
```

Comment:
if `err == nil` then the output here will read `<nil>`. What about
```go
if n, err := rand.Read(m[:]); err != nil {
   t.Fatal(err)
} else if n != len(m) {
   t.Fatal("short read from RNG")
}
```

Reply:
that's right